package.json

```diff
@@ -21,7 +21,8 @@
     "grunt-contrib-jshint": "^1.0.0",
     "grunt-simple-mocha": "~0.4.0",
     "mocha": "^2.0.0",
-    "nconf": "^0.8.4"
+    "nconf": "^0.8.4",
+    "rewire": "^2.5.2"
```

Comment:
`rewire` is required to invalidate `require` cache to verify the requested selenium-version is installed successfully - since unit tests install the new version at run-time.